# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1322690018 28800
#      Wed Nov 30 13:53:38 2011 -0800
# Node ID 62a18e06cfc54335ea452964f23a3b2b68f331e0
# Parent  16b3736e534b130b6780874ebe7d117d919ae822
Fix %!test bug with isprint on MinGW platform (Bug #34338)

mappers.cc (isprint): Don't make character 10 <Return> a printable character
on MinGW.  This makes MinGW compatible with all other Octave platforms.

diff --git a/src/mappers.cc b/src/mappers.cc
--- a/src/mappers.cc
+++ b/src/mappers.cc
@@ -1270,19 +1270,16 @@ are not.\n\
   return retval;
 }
 
 /*
 %!test
 %! charset = char (0:127);
 %! result = zeros (1, 128);
 %! result (33:127) = 1;
-%! if (ispc () && ! isunix ())
-%!   result(10) = 1;
-%! endif
 %! assert(all (isprint (charset) == result));
 
 %!error isprint (1, 2);
 %!error isprint ();
 */
 
 DEFUNX ("ispunct", Fispunct, args, ,
     "-*- texinfo -*-\n\
